doc/docusaurus/docs/using-plutus-tx/cli-plutus.md

```diff
@@ -0,0 +1,347 @@
+---
+sidebar_position: 40
+---
+
+# CLI tool for Plutus
+
+You can locally (without starting a Cardano Node) [run](#running) or [debug](#debugging) compiled code
+by using the `plutus` CLI tool. This tool allows you also to [optimise](#converting) compiled code, [convert](#converting) between code formats,
+and [check](#checking) code for common issues. A pre-built version of the `plutus` CLI executable
+can be found on the [Latest release](https://github.com/IntersectMBO/plutus/releases/latest) page in the repository. Alternatively, you can build the tool
+using Nix, specifically for your platform:
+
+``` shell
+$ nix build .#cabalProject.$(nix eval nixpkgs#stdenv.buildPlatform.system).hsPkgs.plutus-core.components.exes.plutus
```

Comment:
- Without the `$` I can copy-paste this line easier. 
- With double quotes around the argument I am protected from `zsh` mis-interpreting it as a comment.

```suggestion
nix build ".#cabalProject.$(nix eval nixpkgs#stdenv.buildPlatform.system).hsPkgs.plutus-core.components.exes.plutus"
```

Reply:
I suspect that without the `shell` specifier this fenced code block will also have the "Copy To Clipboard" button.